_data/changelogs/component-modal.yml

```diff
@@ -2,6 +2,13 @@ title: Modal
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Updated how the modal JavaScript resets body padding and improved body padding detection.
+  summaryAdditional: Now the component will rely on the CSS to reset body padding rather than injecting an inline style with JavaScript.
+  affectsJavascript: true
```

Comment:
Should we add the `affectsStyles` tag as welll?

Reply:
Good idea. Added in [26fe772](https://github.com/uswds/uswds-site/pull/2273/commits/26fe772822e02e8533a7c2d4ccd51684f9a95116)